### PR TITLE
Api/submit review

### PR DESCRIPTION
**Pull request by kamujusrujan**
@TulasiJagan49  Completed the api/submit_merge. Please check

### CODE REVIEW

File: Project 1/templates/mainpage.html [hunk 1]
```diff
@@ -1,12 +1,42 @@
...
+    <label for="inputPassword2" class="sr-only">author</label>
+    <input type="text" name='author' class="form-control"  placeholder="author">
+  </div>
+  <div class="form-group mx-sm-3 mb-2">
+    <label for="inputPassword2" class="sr-only">year</label>
+    <input type="text" name='year' class="form-control" placeholder="year">
+  </div>
+  <button class="btn btn-primary mb-2">Search</button>
+</form>
+
```
Comment (TulasiJagan49) [inline Project 1/templates/mainpage.html:31]: @kamujusrujan Where did you bind the addrev function written in script.js in this html page? Or addrev is being called from bookdetails page?

Comment (kamujusrujan) [reply Project 1/templates/mainpage.html:31]: it is being called from the book details page. we modularized it so it would be easy when merging

Comment (TulasiJagan49) [reply Project 1/templates/mainpage.html:31]: What I want to know is your addrev function must be called when some one clicks on submit the review button? So, I want to knoe where is that binding?

Comment (kamujusrujan) [reply Project 1/templates/mainpage.html:31]: @TulasiJagan49  there is an another function named review_book. this is the function that's called when we click on submit review button.

Comment (TulasiJagan49) [reply Project 1/templates/mainpage.html:31]: @kamujusrujan Okie

**Review by TulasiJagan49 — APPROVED** [review]
@kamujusrujan You can merge it into master branch after completing the peer reviews.

**Review by sourabh-swargam — APPROVED** [review]

**Pull request merged**